perf/perf_uprobe.py

```diff
@@ -84,34 +85,34 @@ def test_uprobe(self):
             self.fail("perf: sysfs file didn't reflect uprobe events")
         output = self.cmd_verify('perf record -o %s -e probe_perf:main -- '
                                  'perf list' % self.temp_file)
-        if 'samples' not in output.stderr:
+        if 'samples' not in output.stderr.decode("utf-8"):
             self.fail("perf: perf.data file not created")
         output = self.cmd_verify(self.report)
 
     def test_uprobe_return(self):
         output = self.cmd_verify('%s ./uprobe_test doit%%return'
                                  % self.cmdProbe)
-        if 'Added new event' not in output.stderr:
+        if 'Added new event' not in output.stderr.decode("utf-8"):
             self.fail("perf: probe of uprobe return failed")
         # RHEL
-        if self.distro_name == "rhel":
+        if self.distro_name == "rhel" and self.distro_version > "7":
             output = self.cmd_verify('%s__return -- ./uprobe_test'
                                      % self.recProbe)
         else:
             output = self.cmd_verify('%s -aR ./uprobe_test' % self.recProbe)
```

Comment:
just one help this will against older RHEL distro ? and can this is applicable for RHEL based distro like Fedora

Reply:
Yes Praveen this will run for older rhel distro as well